### DISCUSSION TITLE
How to train model including solid-liquid interface in tribochemistry？

### DISCUSSION THREAD

**Discussion by ghost** [deepmodeling/deepmd-kit#1161] [Q&A] [answered]
Dear all,

Currently, I am plan to use DeepMD to study tribochemistry. Typically, the model in tribochemistry involving a slab of lubricant confined between two atomistic wall. I'm not sure whether DeePMD can achieve my goal? If it can, I may need some general advice for training the model. 
Any discussions and suggestions are truly appreciated!

Thanks

Luyao

**Comment by jameswind** [comment]
Sounds like a big project. Do you know any references that use molecular
dynamics to study tribochemistry? How many atoms would need to be simulated?

**Reply by ghost** [reply]
Thank you for your reply. Yes, There are some literatures about tribochemistry using reaxff force field and LAMMPS. Attached is an exmple of simulation model from work by Mohammadtabar et al. (Heat-, Load-, and Shear-Driven Reactions of Di-tert-butyl Disulfide on Fe(100), https://doi.org/10.1021/acs.jpcc.9b05068). According to my experience, I think a typical model should include 10000 atoms.

![1](https://user-images.githubusercontent.com/91039186/133976525-50e84dab-458b-4932-bb46-c7379759fa80.png)
.

**Reply by ghost** [reply]
As far as I know, DeePMD was used to study homogeneous systems and their bulk properties. As to system involving interface, e.g. tribochemistry or wetting , I'm not sure how to prepare the training set. Is it similar to homogeneous system, like a box of water?

**Reply by amcadmus** [reply]
Not really, plz refer to for example https://doi.org/10.1039/C9SC05116C

**Reply by ghost** [reply]
amcadmus, thank you very much! It's very helpful. I will read this paper carefully.

**Comment by jameswind** [comment, marked as answer]
There seem to be more and more works doing inhomogeneous systems. For
example, you may take a look at this one (
https://pubs.rsc.org/en/content/articlehtml/2020/sc/c9sc05116c), or this
one (https://www.science.org/doi/abs/10.1126/science.abd7716), which are
studied by DeePMD simulations.

**Reply by ghost** [reply]
Great! Thank you! I'm more confident to study tribochemistry using DeepMD.
